www/js/diary/infinite_scroll_place_item.js

```diff
@@ -0,0 +1,45 @@
+
+/**
+ * A directive to display each place within the diary view.
+ */
+
+angular.module('emission.main.diary.infscrollplaceitem',
+    ['emission.main.diary.infscrolllist',
+        'emission.survey.multilabel.services',
+        'emission.services',
+        'emission.config.imperial',
+        'emission.config.dynamic',
+        'emission.plugin.logger',
+        'emission.stats.clientstats',
+        'emission.survey.enketo.add-note-button',])
+
+  .directive("infiniteScrollPlaceItem", function () {
+    return {
+      restrict: 'E',
+      scope: {
+        trip: '=',
+        config: '=',
+        surveys: '=',
+      },
+      controller: 'PlaceItemCtrl',
+      templateUrl: 'templates/diary/place_list_item.html'
+    };
+  })
+
+  .controller("PlaceItemCtrl", function ($scope, $state, DynamicConfig) {
+    console.log("Place Item Controller called");
+    console.log('config is ', $scope.config);
+
+    // timebounds is used in js/survey/enketo/enketo-add-note-button.js getPrefillTimes() function
+    // this allows us to pre-fill time and date in surveys that have 'Date', 'Start_time', and 'End_time' fields
+    $scope.timeBounds = () => {
+      let obj = {
+        isPlace: true,
+        enter_fmt_time: $scope.trip?.end_fmt_time,
+        exit_fmt_time: $scope.trip?.nextTrip?.start_fmt_time
```

Comment:
once we start supporting places, we can and should use the place `enter` and `exit` time directly instead of inferring from the trip.